tools/cli/src/mentat_cli/repl.rs

```diff
@@ -52,21 +52,22 @@ impl Repl {
 
     /// Runs the REPL interactively.
     pub fn run(&mut self) {
-        let mut more: Option<Command> = None;
+        let mut more = false;
```

Comment:
I thought I had removed that here, but it turns out it was done elsewhere. Removing...